Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -309,8 +360,11 @@ AActor* ABenchmarkGymGameMode::FindPlayerStart_Implementation(AController* Playe
 	PlayerIdToSpawnPointMap.Add(PlayerUniqueID, ChosenSpawnPoint);
 
 	UE_LOG(LogBenchmarkGym, Log, TEXT("Spawning player %d at %s."), PlayerUniqueID, *ChosenSpawnPoint->GetActorLocation().ToString());
+	
+	AIControlledPlayers.Push(Player);
```

Comment:
Didn't we have some concern at some point that FindPlayerStart could be called multiple times for a single player?

Reply:
Also this says AIControlledPlayers, but wouldn't this get hit for normal clients? Should we check to see if this is a sim player before attempting to add it?

Reply:
The map test above should deal with multiple calls to FindPlayerStart for the same controller